app/models/neighbour.rb

```diff
@@ -7,7 +7,7 @@ class AddressValidationError < StandardError; end
   has_many :neighbour_letters, dependent: :destroy
   has_many :neighbour_responses, dependent: :destroy
 
-  enum source: {manual_add: "manual_add", map_selection: "map_selection", sent_comment: "sent_comment"}
+  enum :source, {manual_add: "manual_add", map_selection: "map_selection", sent_comment: "sent_comment"}
```

Comment:
We can avoid the duplication here by doing this:

``` ruby
enum :source, %w[manual_add map_selection sent_comment].index_by(&:itself)
```

Reply:
That's useful to know, there were a few cases like this I think.

Reply:
Actually, this is better:

``` ruby
enum :source, %i[manual_add map_selection sent_comment].index_with(&:to_s)
```

That gives symbol keys and string values.

Reply:
Naturally we have one enum where the keys use snake case and the values use kebab case, and another where the keys are lowercase and the values uppercase. I'm not sure those are worth trying to deduplicate.